tribestream-api-registry-webapp/src/main/static/assets/scripts/endpoints.ts

```diff
@@ -126,6 +126,9 @@ angular.module('tribe-endpoints', [
                             systemMessagesService.info("Created application details!");
                             let res = saveResponse.data;
                             $location.url(`/application/${res.humanReadableName}?version=${res.swagger.info.version}`);
+                          },
+                          function() {
+                            systemMessagesService.error("Unable to create application.");
```

Comment:
Can you please open up a new ticket to improve exception handling in create application/endpoint and write the ticket here so we know we have some improvements to do. Then we can merge this one.
Don't add it to the board for now please.


Thanks